src/bin/solang_new_examples/solana/flipper.sol

```diff
@@ -0,0 +1,22 @@
+@program_id("F1ipperKF9EfD821ZbbYjS319LXYiBmjhzkkf5a26rC")
+contract flipper {
+	bool private value;
+
+	/// Constructor that initializes the `bool` value to the given `init_value`.
```

Comment:
In Solidity, triple slashes are used with [NatSpec](https://docs.soliditylang.org/en/latest/style-guide.html#natspec). If we use them here we should add a couple of [NatSpec Tags](https://docs.soliditylang.org/en/latest/natspec-format.html#tags) too (alternatively just use normal comments)

Reply:
If not NatSpec tags are used, it defaults to `@notice`, which works fine for this.

Reply:
BTW @salaheldinsoliman just copied the examples which I think are fine to begin with. If we want to add natspec, then it should be done everywhere in the repo (I don't think it is necessary)

Reply:
FWIW I think NatSpec is pointless garbage. Rust does not have it and it is better for it.